### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by mdandidarmawan**
Adding:
- **Table of contents**
- **Contributing** link section

Updating:
- **License / Usage** to **License**

### CODE REVIEW

**Comment by voodoods** [comment]
Nice work! 😃 

I'd like to wait for https://github.com/voodoods/webpack-boilerplate/issues/5 to be finished, so we can add that to the `README.md` as well. If you like you can go ahead and already add the link to the upcoming `CODE_OF_CONDUCT.md` file 👍

**Comment by mddarmawan** [comment]
Done! :smile:

File: README.md [hunk 1]
```diff
@@ -16,5 +28,11 @@ This setup comes with the follwing features:
 - ES6 Support via [babeljs](https://babeljs.io/)
 - An EditorConfig setup
 
-## License / Usage
+## License
 Feel free to fork this repo and modify it according to your needs. Leave me a note if you like this setup.
+
+## Code of Conduct
+This repository contains the Open Source [Code of Conduct](code-of-conduct.md).
```
Comment (voodoods) [inline README.md]: The file is now in the master branch: https://github.com/voodoods/webpack-boilerplate/blob/master/CODE_OF_CONDUCT.md

Please adjust the filename accordingly, then we're set 👍

**Comment by mddarmawan** [comment]
Done :smile:

**Pull request closed without merging**